buildSrc/src/main/java/com/google/firebase/gradle/ReleaseGenerator.kt

```diff
@@ -40,12 +57,35 @@ open class ReleaseGenerator : DefaultTask() {
     val branchRef = getObjectRefForBranchName(repo, pastRelease)
 
     val changedLibraries = getChangedLibraries(repo, branchRef, headRef, firebaseLibraries)
+    val changes = getChangesForLibraries(repo, branchRef, headRef, changedLibraries)
```

Comment:
Since this only gets used in the case of `printReleaseConfig`, we could save some overhead by moving it down to where the if statement is. Also worth noting that you can do something like this too:
```kotlin
if (printReleaseConfig) {
  getChangesForLibraries(repo, branchRef, headRef, changedLibraries).also {
    println(generatePrintOutput(it))
  }
}
```

Or if you wanted to be extra:

```kotlin
if (printReleaseConfig) {
  getChangesForLibraries(repo, branchRef, headRef, changedLibraries).let {
    generatePrintOutput(it)
  }.also {
    println(it)
  }
}
```